### DISCUSSION TITLE
Remote controller(88010) is not conntecd with Technic Hub

### DISCUSSION THREAD

**Discussion by sangb-han** [pybricks/support#672] [Q&A]
Before full dive into pybrick, i have tested in a project in pybrics website. 
Off load buggy in following location was my target. 

https://pybricks.com/projects/sets/technic/42124-off-road-buggy/powered-up-remote/

When I tested it in web IDE, remote controller was connected perfectly. 
When I download the code into technic hub. i.e. offline mode, remote conrtroller was not connected. 

So i moved into beta.pybricks.com. Almost same. 
Work perfectly in web IDE. 
In offlinemode, mostly paring failed and responsiveness of succssful paring was very slow. 
Maybe, one of three tries was successful. 

How do i debug? or Anything that i try?

**Comment by laurensvalk** [comment]
Thanks for reporting. Can you try the following in offline mode?

- Turn on the hub (1 button click on hub).
- Wait at least 10 seconds.
- Start the remote (1 button click on remote).
- Start the program (1 button click on hub.)

So everything is the same as usual, but you wait a bit longer for the hub and the devices to fully boot up.

Is it reliable this way?

If so, I think this might be the underlying cause: https://github.com/pybricks/support/issues/673

**Reply by sangb-han** [reply]
@laurensvalk  I 'll follow your sequence and report the result. :) 
Thank you .

**Reply by sangb-han** [reply]
@laurensvalk I checked it and it worked perfectly. 

In my thought, after the sync and installation of program is completed, visual signal is required. For example, cycling color of led.

**Reply by laurensvalk** [reply]
Glad to hear that it worked! And I agree, the current behavior is not intuitive.

After you first posted the issue, I started digging and identified this issue as the root cause: https://github.com/pybricks/support/issues/674

Once we fix that, the 10 second wait will no longer be required. This should be fixed by the time we do another release.
